app/views/shared/check_answers/_relationship_to_child.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= govuk_summary_list do |summary_list|
+      summary_list.with_row do |row|
+        row.with_key(text: t(".biological_parent"), classes: "govuk-!-width-one-half")
+        row.with_value(text: yes_no(@relationship_to_child.eql?("biological")))
+        row.with_action(text: "Change", href: providers_merits_task_list_is_client_biological_parent_path(@proceeding), visually_hidden_text: t(".biological_parent"))
+      end
+
+      unless @relationship_to_child.eql?("biological")
+        summary_list.with_row do |row|
+          row.with_key(text: t(".parental_responsibilities"))
+          row.with_value(text: @relationship_to_child.nil? ? t("generic.no") : t(".#{@relationship_to_child}"))
+          row.with_action(text: "Change", href: providers_merits_task_list_does_client_have_parental_responsibility_path(@proceeding), visually_hidden_text: t(".parental_responsibilities"))
```

Comment:
Should the text be a translation ie t("generic.change")

Reply:
Yep, good spot